official/mnist/mnist.py

```diff
@@ -30,68 +30,59 @@
 LEARNING_RATE = 1e-4
 
 
-class Model(tf.keras.Model):
+def create_model(data_format):
   """Model to recognize digits in the MNIST dataset.
 
   Network structure is equivalent to:
   https://github.com/tensorflow/tensorflow/blob/r1.5/tensorflow/examples/tutorials/mnist/mnist_deep.py
   and
   https://github.com/tensorflow/models/blob/master/tutorials/image/mnist/convolutional.py
 
-  But written as a tf.keras.Model using the tf.layers API.
-  """
+  But uses the tf.keras API.
+
+  Args:
+    data_format: Either 'channels_first' or 'channels_last'. 'channels_first' is
+      typically faster on GPUs while 'channels_last' is typically faster on
+      CPUs. See
+      https://www.tensorflow.org/performance/performance_guide#data_formats
```

Comment:
This is neat. Anticipating that people are going to ask how this is different than the Resnet/__call__ style, and why we use tf.keras.layers here, can you add comments explaining what Sequential does? And, for our reference-- should we be using tf.keras.layers at this point? Sequential in the cases where it makes sense?

Reply:
`Sequential` is a sub-class of `Model` so this isn't really different, it's just simpler. If your network is really just a chain of layers, `Sequential` saves some boilerplate.

I'm a bit hesitant to add a comment here explaining that since that comment would either have to pre-suppose that model-subclassing is the most typical usage (which it isn't), or first explain that there is another more-verbose way to write this network and then explain how `Sequential` is simpler :). But perhaps I'm overthinking, so if you have a suggestion of what the comment should be - happy to add it.

Yes, in other samples we should use `Sequential` wherever it fits, and use `tf.keras.layers`.

Reply:
"A subclass of tf.keras.Model, tf.keras.Sequential, is returned here, as the MNIST model is just a chain of layers." Or something to that effect.

Reply:
Done